VideoApp/VideoApp/ViewControllers/Room/RoomViewModel.swift

```diff
@@ -91,12 +97,21 @@ class RoomViewModel {
         guard let payload = notification.payload as? Room.Update else { return }
         
         switch payload {
-        case .didStartConnecting, .didConnect: delegate?.didConnect()
-        case let .didFailToConnect(error): delegate?.didFailToConnect(error: error)
-        case let .didDisconnect(error): delegate?.didDisconnect(error: error)
-        case .didStartRecording: delegate?.didUpdateRecording()
-        case .didStopRecording: delegate?.didUpdateRecording()
-        case .didAddRemoteParticipants, .didRemoveRemoteParticipants, .didUpdateParticipants: break
+        case .didStartConnecting:
+            delegate?.didConnect()
```

Comment:
I don't think this is causing a bug but it is confusing. I probably renamed something and didn't realize this was confusing. I will create a separate ticket and plan to fix in this sprint.

Reply:
https://issues.corp.twilio.com/browse/VIDEO-4203